atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/StructureHolder.java

```diff
@@ -23,7 +23,7 @@
  * Utility class to make manipulating {@link io.vavr.collection.Map} and {@link io.vavr.collection.Set} less
  * error-prone as all methods that modify state return a *new* instance in vavr.
  */
-final class StructureHolder<V> {
+public final class StructureHolder<V> {
```

Comment:
I'd imagine this was done in the first place to avoid ABI breaks, but I need something similar in the workload server! If we are against making this public, then will re-create it in the workload server 😬

Reply:
I'm fine with this (it is in the shared project).